browser/ui/views/tabs/brave_tab_container.cc

```diff
@@ -570,6 +570,18 @@ void BraveTabContainer::UpdateScrollBarVisibility() {
   NOTIMPLEMENTED();
 }
 
+bool BraveTabContainer::HandleVerticalScroll(int y_offset) {
+  if (tabs::utils::ShouldShowVerticalTabs(tab_slot_controller_->GetBrowser()) &&
+      IsUnpinnedTabScrollEnabled() && y_offset != 0) {
```

Comment:
Checking `IsUnpinnedTabScrollEnabled()` is necessary?
It gives same value with `tabs::utils::ShouldShowVerticalTabs()`.

Reply:
Yes, actually, IsUnpinnedTabScrollEnabeld() could be entirely removed.